cirq/ops/fsim_gate.py

```diff
@@ -301,7 +306,26 @@ def rz_angles_after(
         a1 = (-self.gamma - self.zeta + self.chi) / 2.0
         return a0, a1
 
+    def _zeta_insensitive(self) -> bool:
+        return self.theta in (-np.pi / 2, np.pi / 2, -sympy.pi / 2,
+                              sympy.pi / 2)
+
+    def _chi_insensitive(self) -> bool:
+        return self.theta in (-np.pi, 0.0, np.pi, -sympy.pi, sympy.pi)
```

Comment:
Maybe we can add module-level helper functions:
```python
def _zero_mod_pi(param: Union[float, sympy.Basic]) -> bool:
    """Returns true if param == 0 (mod pi)."""
    return param in (-np.pi, 0.0, np.pi, -sympy.pi, sympy.pi)

def _half_pi_mod_pi(param: Union[float, sympy.Basic]) -> bool:
    """Returns true if param == pi / 2 (mod pi)."""
    return param in (-np.pi / 2, np.pi / 2, -sympy.pi / 2, sympy.pi / 2)

...
def _zeta_insensitive(self) -> bool:
    return _half_pi_mod_pi(self.theta)

def _chi_insensitive(self) -> bool:
    return _zero_mod_pi(self.theta)

def qubit_index_to_equivalence_group_key(self, index: int) -> int:
    if ((_zero_mod_pi(self.zeta) or self._zeta_insensitive()) and
        (_zero_mod_pi(self.chi) or self._chi_insensitive())):
         ...
```

Reply:
Done.